src/components/DateTimePicker/DateTimePicker.jsx

```diff
@@ -263,36 +259,34 @@ const DateTimePicker = ({
     ...i18n,
   };
 
+  // State
   const [isExpanded, setIsExpanded] = useState(expanded);
   const [customRangeKind, setCustomRangeKind] = useState(
     showRelativeOption ? PICKER_KINDS.RELATIVE : PICKER_KINDS.ABSOLUTE
   );
-
   const [isCustomRange, setIsCustomRange] = useState(false);
-
-  const datePickerRef = React.createRef();
-
   const [selectedPreset, setSelectedPreset] = useState(null);
-
   const [currentValue, setCurrentValue] = useState(null);
   const [lastAppliedValue, setLastAppliedValue] = useState(null);
   const [humanValue, setHumanValue] = useState(null);
-
   const [relativeValue, setRelativeValue] = useState(null);
   const [absoluteValue, setAbsoluteValue] = useState(null);
-
   const [focusOnFirstField, setFocusOnFirstField] = useState(true);
 
+  // Refs
+  const datePickerRef = React.createRef();
+  const relativeSelect = React.createRef(null);
+
   const dateTimePickerBaseValue = {
     kind: '',
     preset: {
-      label: null,
-      offset: null,
+      label: presets[0].label,
+      offset: presets[0].offset,
     },
     relative: {
       lastNumber: null,
-      lastInterval: null,
-      relativeToWhen: null,
+      lastInterval: intervals[0].value,
+      relativeToWhen: relatives[0].value,
```

Comment:
Should the new defaults be set in defaultProps instead of functionally applying defaults here? I think the primary benefit to formally declaring in the defaultProps would be for the automatic documentation we have in storybook.

Reply:
These are not defaults necessarily there is no preset, relative prop. These are arrays and we are setting the first item in the array as the base value.

Reply:
We were initializing everything as null but there are times when we need to calculate the value and an onchange of these values has not yet occurred. Defaulting to the first item in the array to avoid the component blowing up.